module Main where

import System.Exit (exitFailure)

-- foo

main = do
    putStrLn "This test always passes!"
    exitFailure